src/kendo.rating.js

```diff
@@ -0,0 +1,308 @@
+/* jshint browser: true, jquery: true */
+/* globals define: false */
+
+(function (f, define) {
+    'use strict';
+    define(['./vendor/kendo/kendo.binder'], f);

```

Comment:
You should change this definition to the one like this:

https://github.com/telerik/kendo-ui-core/blob/master/src/kendo.button.js#L2